src/components/victory-pie.jsx

```diff
@@ -177,16 +181,16 @@ export default class VictoryPie extends React.Component {
 
   getStyles(props) {
     const style = props.style || defaultStyles;
-    const {data, labels, parent} = style;
+    const { data, labels, parent } = style;
     return {
-      parent: _.merge({height: props.height, width: props.width}, parent),
-      data: _.merge({}, defaultStyles.data, data),
-      labels: _.merge({}, defaultStyles.labels, labels)
```

Comment:
I might roll it back, for now. Thought I would do two things at once...

See more about ^ here https://twitter.com/matzatorski/status/685551690205007872